config/default/kustomization.yaml

```diff
@@ -3,7 +3,7 @@ namespace: garo-system
 
 # Value of this field is prepended to the
 # names of all resources, e.g. a deployment named
-# "wordpress" becomes "alices-wordpress".
+# "wordpress" becomes "garo-wordpress".
```

Comment:
The change isn't really required, nor is it consistent. It's a generated explanation of how it works.